haystack/telemetry/_environment.py

```diff
@@ -12,6 +12,10 @@
 
 logger = logging.getLogger(__name__)
 
+# Any remote API (OpenAI, Cohere etc.)
+HAYSTACK_REMOTE_API_BACKOFF_SEC = "HAYSTACK_REMOTE_API_BACKOFF_SEC"
+HAYSTACK_REMOTE_API_MAX_RETRIES = "HAYSTACK_REMOTE_API_MAX_RETRIES"
+HAYSTACK_REMOTE_API_TIMEOUT_SEC = "HAYSTACK_REMOTE_API_TIMEOUT_SEC"
```

Comment:
Where are these used?

Reply:
This was from an earlier attempt to solve the issue, as it is done in v.1x but it must have slipped when I reversed all changes.